mmv1/templates/terraform/constants/cloud_run_domain_mapping.go.erb

```diff
@@ -1,6 +1,6 @@
 var domainMappingGoogleProvidedLabels = []string{
```

Comment:
I think we're able to put the entire DSF + variable here into the state upgrader file, since that should now be the only user, not just the strings themselves!